Neos.Neos/Resources/Public/JavaScript/Interfaces/NeosConfiguration.ts

```diff
@@ -0,0 +1,5 @@
+export default interface NeosConfiguration {
+  init: () => void;
+  get: (key: string) => string;
+  override: (key: string, value: any) => void;
```

Comment:
Probably something else than `any` would bet better as we have strings, numbers, array, objects. But I can live with it.